tools/db/servervars.php

```diff
@@ -11,45 +11,71 @@
   if (!isset($mysqluser))
     $mysqluser = isset($_SERVER['api_keyman_com_mssql_user']) ? $_SERVER['api_keyman_com_mssql_user'] : null;
 
-  if (!isset($mssqldb0)) $mssqldb0 = $_SERVER['api_keyman_com_mssqldb0'];
-  if (!isset($mssqldb1)) $mssqldb1 = $_SERVER['api_keyman_com_mssqldb1'];
+  if (!isset($mssqldb)) $mssqldb = $_SERVER['api_keyman_com_mssqldb'];
   if (!isset($mssqlconninfo)) $mssqlconninfo = $_SERVER['api_keyman_com_mssqlconninfo'];
-  if (!isset($mssql_create_databases) && isset($_SERVER['api_keyman_com_mssql_create_databases'])) $mssql_create_databases = $_SERVER['api_keyman_com_mssql_create_databases'];
+  if (!isset($mssql_create_database) && isset($_SERVER['api_keyman_com_mssql_create_database']))
+    $mssql_create_database = $_SERVER['api_keyman_com_mssql_create_database'];
 
-  class ActiveDB
+  class DatabaseConnectionInfo
```

Comment:
Further refactoring to use `DatabaseConnectionInfo` will occur in a follow-up PR